scrapi/harvesters/scholarsarchiveosu.py

```diff
@@ -0,0 +1,29 @@
+'''
+Harvester for the ScholarsArchive@OSU for the SHARE project
+
+Example API call: http://ir.library.oregonstate.edu/oai/request?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import helpers
+from scrapi.base import OAIHarvester
+
+
+class ScholarsarchiveosuHarvester(OAIHarvester):
+    short_name = 'scholarsarchiveosu'
+    long_name = 'ScholarsArchive@OSU'
+    url = 'http://ir.library.oregonstate.edu/oai/request'
```

Comment:
I don't think this is the correct url for the service (should point to a landing page or something)